sdk/samples/storage/blobs/README.md

```diff
@@ -52,11 +52,11 @@ Azure SDK for Embedded C is licensed under the [MIT][azure_sdk_for_c_license] li
 <!-- LINKS -->
 [azure_sdk_for_c_contributing]: https://github.com/Azure/azure-sdk-for-c/blob/main/CONTRIBUTING.md
 [azure_sdk_for_c_license]: https://github.com/Azure/azure-sdk-for-c/blob/main/LICENSE
-[BLOB_SDK_README]: https://github.com/Azure/azure-sdk-for-c/tree/main/sdk/docs/storage
-[SDK_README_CONTRIBUTING]:https://github.com/Azure/azure-sdk-for-c/tree/main/sdk/docs/storage#contributing
```

Comment:
Btw, this link is not being referenced anywhere in the markdown and can be remove (unrelated cleanup, we can do later)